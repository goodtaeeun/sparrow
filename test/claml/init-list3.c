typedef struct {
  unsigned int a;
} my_type;
my_type myType = {&myType};
int main() { return 0; }
