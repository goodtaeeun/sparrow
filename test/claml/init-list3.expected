/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "init-list3.c"
struct __anonstruct_0 {
  unsigned int a;
};
#line 3 "init-list3.c"
typedef struct __anonstruct_0 my_type;
#line 4 "init-list3.c"
my_type myType = {(unsigned int)(&myType)};
#line 5 "init-list3.c"
int main(void) {

  {
#line 5
    return (0);
  }
}
